CHANGELOG.md

```diff
@@ -15,7 +15,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
     _Security_ in case of vulnerabilities.
  -->
 
-## [Unreleased](https://github.com/o1-labs/o1js/compare/6ff7f8470a...HEAD)
+## [Unreleased](https://github.com/o1-labs/o1js/compare/4e23a60...HEAD)
+
+## [2.5.0](https://github.com/o1-labs/o1js/compare/6ff7f8470a...4e23a60)
```

Comment:
There's two Added and Changed sections under this release

Reply:
Weird... but fixed